dotnet/src/Connectors/Connectors.OpenAI.UnitTests/Core/OpenAIJsonSchemaTransformerTests.cs

```diff
@@ -16,10 +16,12 @@ public sealed class OpenAIJsonSchemaTransformerTests
 {
     private static readonly AIJsonSchemaCreateOptions s_jsonSchemaCreateOptions = new()
     {
-        IncludeSchemaKeyword = false,
```

Comment:
@RogerBarreto for context:

* `IncludeSchemaKeyword` is false by default.
* `IncludeTypeInEnumSchemas` has been removed since the type keyword is always being included.
* `MoveDefaultKeywordToDescription` is a transformation that used to be tied to `RequireAllProperties` and is now toggled independently.